.changes/next-release/feature-HTTPclientspi-25d1ed9.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "HTTP client spi",
```

Comment:
Please capitalize here to `HTTP Client SPI` to be consistent with the rest of the log entries

Reply:
I made an educated guess before so was unaware of the casing significance